### ISSUE TITLE
Support creation of a remark slideshow

### ISSUE THREAD

**Issue by sualeh** [mike-ward/Markdown-Edit#137]
[Mike](@mike-ward), it would be great if you could support the creation of a [remark](https://github.com/gnab/remark) slideshow, with a preview and export of the slideshow in Markdown-Edit.

**Comment by RoyiAvital** [comment]
Wow! Great idea.

First we take Word then we take Power Point :smile: .

**Comment by veremenko-jav** [comment]
+1

**Comment by mike-ward** [comment]
Finally got around to looking at remark.js. Since it's just an HTML file with what looks like a Markdown island embedded in it I propose following.

If a document starts with

    <!DOCTYPE html>

Preview will not load the default template and instead simply render the document.

What do you think? Am I missing something?

**Comment by mike-ward** [comment]
I fooled around with modifying MDE to handle HTML straight-up. Bottom line, it really isn't suited to the task.

I did get a remark slide show to work but it was awkward. The preview was slow because the everything is rendered in browser. Other than color highlighting, there was not much MDE offered to assist in writing slides in remark. I found it easier to use VS Code and keep a browser window open and just refresh it as I added slides. 

Unless someone has some other insights I'm going to drop the idea of slide generation in MDE.
